src/Microsoft.AspNetCore.Mvc.Testing/WebApplicationFactory.cs

```diff
@@ -49,14 +49,13 @@ public class WebApplicationFactory<TEntryPoint> : IDisposable where TEntryPoint
         /// <typeparamref name="TEntryPoint" /> will be loaded as application assemblies.
         /// </para>
         /// </summary>
-        public WebApplicationFactory() : 
-            this(builder => { }, new WebApplicationFactoryClientOptions())
+        public WebApplicationFactory() : this(new WebApplicationFactoryClientOptions())
         {
+            _configuration = ConfigureWebHost;
         }
 
-        private WebApplicationFactory(Action<IWebHostBuilder> configuration, WebApplicationFactoryClientOptions options)
+        private WebApplicationFactory(WebApplicationFactoryClientOptions options)
```

Comment:
Suggest either removing this constructor or using it from `DelegatedWebApplicationFactory`. As-is, the separation is only confusing.

Reply:
The `DelegatedWebApplicationFactory` constructor overwrites `_configuration` and `ClientOptions`. Since everything works with it using the parameterless constructor, my preference is to remove this split.